timeline-chart/src/layer/time-graph-chart.ts

```diff
@@ -79,6 +79,9 @@ export class TimeGraphChart extends TimeGraphChartLayer {
         }
 
         const moveVertically = (magnitude: number) => {
+            if (this.rowController.totalHeight <= this.stateController.canvasDisplayHeight) {
```

Comment:
Somehow this doesn't prevent scrolling if the total height of the tree has been reduced by collapsing some tree elements. Should the totalHeight be recalculated?

Reply:
Looks like the totalHeight is set by the user of the timeline-chart, so it would need to update it when a node is toggled.

Reply:
Thanks for investigating this.